template/build/webpack.base.conf.js

```diff
@@ -16,10 +16,12 @@ module.exports = {
     extensions: ['', '.js', '.vue'],
     fallback: [path.join(__dirname, '../node_modules')],
     alias: {
-      {{#if_eq build "standalone"}}
-      // nessessary to make "import Vue from 'vue'" load the standalone version.
+      {{#if_eq build "standalone"}}∞
       'vue': 'vue/dist/vue',
       {{/if_eq}}
```

Comment:
That's a good point! Then maybe we should merge this even before a decision is made, since it future-proofs users of the 2.0 branch against any possible changes of the default export.